chart/values.yaml

```diff
@@ -28,6 +28,10 @@ namespaceOverride: ""
 ##
 fullnameOverride: ""
 
+ssl-exporter:
+  enabled: false
+  serviceMonitor:
+    enabled: true
```

Comment:
Why do we need to have serviceMonitor enabled for `ssl-exporter` when it is disabled?

Reply:
Because if we enable ssl-exporter, it's assumed we also want the service monitor, since we already have prometheus in our monitoring stack.